src/aux-server/aux-web/shared/vue-components/ImageClassifier/ImageClassifier.ts

```diff
@@ -55,8 +57,9 @@ export default class ImageClassifier extends Vue {
     modelJsonUrl: string = null;
     modelMetadataUrl: string = null;
     cameraType: CameraType = null;
+    image: Image[] = null;
 
-    private _openEvent: OpenImageClassifierAction;
+    private _openEvent: OpenImageClassifierAction | OpenClassifyImagesAction;
```

Comment:
@TroyceGowdy `_openEvent` never gets assigned a `OpenClassifyImagesAction`, so its type definition can be simplified.